minutes/2021-10-05.md

```diff
@@ -0,0 +1,35 @@
+# Parsec Community Meeting (05/10/2021) 📋
+
+Minutes of previous meeting are stored
+[here](https://github.com/parallaxsecond/community/tree/main/minutes).
+
+## Attendance
+
+- Anton Antonov
+- Paul Howard
+- Wiktor Kwapisiewicz
+- Marc Meunier
+- Ionut Mihalcea
+
+## Agenda
+
+- Discuss roadmap for v0.9.0:
```

Comment:
I think the next version should be `0.8.x` and not `0.9.0` to show that we are not performing any breaking change.